### DISCUSSION TITLE
iop Editor

### DISCUSSION THREAD

**Discussion by Thom-de-Jong** [Open-Agriculture/AgIsoStack-plus-plus#34] [General]
What object pool editor do you use?

I am currently using the demo version of Jetter ISO-Designer.

**Comment by ad3154** [comment]
I'm also currently using the demo of Jetter ISO-Designer.

I have tried this before https://github.com/moehman/PoolEditDist, but had troubles on windows 11. And the interface is not as nice as Jetter's.
